category-search-page/category-search-page.js

```diff
@@ -0,0 +1,46 @@
+let container = document.querySelector('.container')
+let input= document.querySelector('.input-value')
+let buttonSearch= document.querySelector('.butt-search')
+let categoryName= localStorage.getItem("categoryName")
```

Comment:
the key of the category name is 'CategoryName' not 'categoryName'
the return from local storage is null

<img width="790" alt="Screen Shot 2022-03-22 at 2 47 40 PM" src="https://user-images.githubusercontent.com/47992412/159485352-092be03b-4659-4293-b209-f58cfd78f5e9.png">


```suggestion
let categoryName= localStorage.getItem("CategoryName")
```

<img width="514" alt="Screen Shot 2022-03-22 at 2 48 41 PM" src="https://user-images.githubusercontent.com/47992412/159485663-a7efadda-ab04-4baa-8848-329fda248f7c.png">